lib/vision/index.js

```diff
@@ -194,6 +194,8 @@ Vision.prototype.annotate = function(requests, callback) {
  *     `logos`, `properties`, `safeSearch`, `text`.
  * @param {boolean=} options.verbose - Use verbose mode, which returns a less-
  *     simplistic representation of the annotation (default: `false`).
+ * @param {object=} options.imageContext - An `ImageContext`. See an
```

Comment:
Can you change this to simply:

```js
 * @param {object=} options.imageContext - See an
       [`ImageContext`](https://cloud.google.com/vision/reference/rest/v1/images/annotate#ImageContext)
       resource.
```

Also, if you wouldn't mind, could you move these lines above `options.maxResults` to keep the arguments alphabetized?